src/website/shared/models/cve.py

```diff
@@ -344,6 +352,49 @@ def generate_code(
         instance.save()
 
 
+# Tracking many-to-many cve relationship of NixpkgsIssue
+@pghistory.track(
+    pghistory.InsertEvent("cve.add"),
+    pghistory.DeleteEvent("cve.remove"),
+    model_name="NixpkgsIssueCveLog",
+)
+class NixpkgsIssueCve(NixpkgsIssue.cve.through):
+    class Meta:
+        proxy = True
+
+
+# Tracking many-to-many derivations relationship of NixpkgsIssue
+@pghistory.track(
+    pghistory.InsertEvent("derivations.add"),
+    pghistory.DeleteEvent("derivations.remove"),
+    model_name="NixpkgsIssueDerivationsLog",
+)
+class NixpkgsIssueDerivation(NixpkgsIssue.derivations.through):
+    class Meta:
+        proxy = True
+
+
+class NixpkgsIssueLogView(models.Model):
+    class Meta:
+        managed = False
```

Comment:
Can we add a comment on why this is not managed by the ORM?

Reply:
I pasted the wrong commit message in the PR description. I'll add similar comments in the model + the fact that Django doesn't seem to support multi table proxy models (which would be the equivalent of VIEW tables in ORM).

Reply:
comment added